src/code/main.c

```diff
@@ -25,6 +43,9 @@ StackEntry sSchedStackInfo;
 StackEntry sAudioStackInfo;
 StackEntry sPadMgrStackInfo;
 StackEntry sIrqMgrStackInfo;
+#if PLATFORM_N64
+uintptr_t gSegments[NUM_SEGMENTS];
+#endif
```

Comment:
`gSegments` moving is pretty suspicious, my instinct is it's bss reordering rather than an actual change. I suggest leaving it where it was before and then we can deal with bss once the rest of the ROM is matched